prow/config/plugins.yaml

```
@@ -118,6 +118,16 @@ blockades:
       This PR modifies the files under the docs folder and requires the docs team to follow up on the PR.
 
       /label docs
+  - repos:
+      - pingcap/tidb
+    blockregexps:
+      - ^sessionctx/variable/
+      - ^config/config.toml.example$
+    explanation: |
+      This PR modifies the config files and requires the config review team to follow up on this PR.
+      You can request the following members of the config review team to review through the `/cc` command:
+
+      @dongyuusa, @bb7133, @XuHuaiyu, @cfzjywxk, @morgo
```

```suggestion
      This PR modifies significant config files which require review from domain experts from anyone of
      
      @dongyuusa, @bb7133, @XuHuaiyu, @cfzjywxk, or @morgo
      
      You can request review from anyone of them by `/cc @reviewer`
```

cc @zhangyangyu please also take a look on the expression.